app/routes.php

```diff
@@ -95,10 +97,16 @@ function( Request $request ) use ( $ffFactory ) {
 		] );
 
 		$responseModel = $useCase->addSubscription( $subscriptionRequest );
-		// TODO forward/dispatch to matching 'page/name' route, depending on $responseModel->getType();
-		return 'TODO';
+		if ( in_array( 'application/json', $request->getAcceptableContentTypes() ) ) {
+			return $app->json( $ffFactory->newAddSubscriptionJSONPresenter()->present( $responseModel ) );
+		}
+		if ( $responseModel->isSuccessful() ) {
+			return $app->redirect( $app['url_generator']->generate('page', [ 'pageName' => 'SubscriptionSuccess' ] ) );
```

Comment:
Maybe we should forward the request using `$app->handle()`? Doing it like this will send 302 to the browser, which then re-requests.